src/index.ts

```diff
@@ -130,3 +130,7 @@ export {IProps as TilesProps} from './modules/tiles/tiles.types';
 
 // Utilities
 export {DeviceUtils, LayoutUtils} from './modules/utilities';
+
+// People-list-item
+export {default as PeopleListItem} from './modules/people-list/people-list-item';
+export {IProps as IPeopleItemProps} from './modules/people-list/people-list-item.type';
```

Comment:
let's make the sub extension here plural, i.e. `types` vs `type` 
thanks

Reply:
I have change type to types